paddle/fluid/operators/neg_op.cu

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/operators/neg_op.h"
+#include "paddle/fluid/platform/float16.h"
+
+namespace ops = paddle::operators;
```

Comment:
Could we consider a CUDAKernel implementation to speed up the calculation in the future?

Reply:
I write a cuda kernel for forward process of neg op.